src/glossary.md

```diff
@@ -0,0 +1,281 @@
+# Fuel Glossary
+
+## Address
+
+An address is a cryptographic hash representing an identity of either a wallet, contract address, or a predicate root.
+
+## AssetId
+
+An AssetId is a unique identifier for an on-chain asset. It is defined as the root of the bytecode of the contract minting the asset.
+
+## Base Asset
+
+The base asset is the underlying asset needed to perform any transactions on a blockchain. It is used to pay gas for transactions. On Ethereum, the base asset is ETH.
+
+## Block
+
+A block is a record of many transactions, that are grouped together and cryptographically hashed. The linked blocks form a chain, also called a blockchain.
+
+## Block Explorer
+
+A block explorer is an interface for block and transaction data produced by a blockchain. You can use a block explorer to explore and verify addresses, contracts, and transaction histories.
+
+## Block Height
+
+The block height refers to the total number of blocks produced in a blockchain.
```

Comment:
I like the definition from [here](https://academy.binance.com/en/glossary/block-height):
"Block height represents the number of blocks that were confirmed in the entire history of a particular blockchain network - from the [genesis block](https://academy.binance.com/en/glossary/genesis-block) (or block zero) until the most recent one."

So maybe something similar? Specifically, I like that it includes a "time" aspect ("until the most recent one")